lib/storage-handler.js

```diff
@@ -51,13 +56,18 @@ exports.upload = function (provider, req, res, container, cb) {
       type: part.mime
     };
 
+    if ('function' === typeof options.renamer) {
+      file.name = options.renamer(file);
```

Comment:
We need to pass the req/res objects to *renamer* so we can name the file depending on the request